app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt

```diff
@@ -97,6 +98,13 @@ class ChatActivity : AppCompatActivity(), IChatView {
         chatPresenter.setUp()
         chatPresenter.checkPreferences()
         setEditText()
+        rv_chat_feed.setOnTouchListener(object : OnSwipeListener(this) {
+            override fun onSwipeRight() {
+                val i = Intent(context, SkillsActivity::class.java)
+                startActivity(i)
```

Comment:
i can be replaced by 'intent' . Although it doesn't make much difference whether we use i or intent here, but it's always good to use mnemonics rather than using variables that tell nothing about the purpose for which the variable has been used.